### DISCUSSION TITLE
Line height font setting

### DISCUSSION THREAD

**Discussion by Superneutrino** [SilenceLeo/snowb-bmf#57] [Ideas]
Hi
I was wandering is there any reason why line height setting is not exposed in editor?

`info face="Font" size=100 bold=0 italic=0 charset="" unicode=1 stretchH=100 smooth=1 aa=1 padding=22,22,22,22 spacing=3,3
common lineHeight=100 base=70 scaleW=1460 scaleH=1475 pages=1 packed=0
page id=0 file="Font.png"`

Maybe developers in my team could help with that since we're using your nice editor for our games.

**Comment by SilenceLeo** [comment]
Hi,

The current row height is automatically filled according to the font.

I think the control of line height should be in the renderer, rather than generating a set of fonts for each line height.

What is the scenario that requires this function?

**Reply by Superneutrino** [reply]
Scenario is simple. Automatic line height sometimes is to big for sentences with two lines of text or more.  Other Bitmap font editors have line height setting like BmGlyph or Glyph Designer for example. 

I assume line height can mess up font position in it's bounding box? (it happened to me)  More about it here.
https://github.com/Gamua/Starling-Framework/issues/902

**Comment by SilenceLeo** [comment]
I added line height settings. https://github.com/SilenceLeo/snowb-bmf/commit/ae18566da7c3f429f2a0b5ac3769fa4523498b19

<img width="267" alt="image" src="https://github.com/SilenceLeo/snowb-bmf/assets/4632034/d7770d43-9fd4-4b97-bca4-78798ca267fd">

**Reply by Superneutrino** [reply]
Thank you for your great work.
